src/ferc_xbrl_extractor/taxonomy.py

```diff
@@ -116,8 +119,36 @@ def from_list(cls, concept_list: list, concept_dict: ConceptDict) -> "Concept":
             child_concepts=[
                 Concept.from_list(concept, concept_dict) for concept in concept_list[3:]
             ],
+            metadata=Metadata.from_concept(concept),
         )
 
+    def get_metadata(
+        self, period_type: Literal["duration", "instant"]
+    ) -> dict[str, dict[str, Any]]:
+        """Get metadata from all leaf nodes in Concept tree.
+
+        Leaf nodes in the Concept tree will become columns in output tables. This method
```

Comment:
If you use the `glossary::` directive in the docs, I think we can also link to the definitions there in the docstrings using e.g. :term:\`concept\`

Reply:
Added links